.markdownlint.yaml

```diff
@@ -0,0 +1,42 @@
+# MD001/heading-increment/header-increment
```

Comment:
Thank you. This will help out a lot. Can you also add a description at the top of the file that notes where these rules come from and how to research them (link to docs)?

Reply:
Sure!

Reply:
✅